### ISSUE TITLE
Processing no running at full speed

### ISSUE THREAD

**Issue by Nematosis** [processing/processing#5876]
## Description

Processing no running at full speed

## Expected Behavior

Processing used to work well for the first months, now whenever I run a sketch it wont go faster than 10 fps no matter how ridiculously small the script is.

If one could actually set the priority of processing within the gpu or at the very least see it, would be helpfull.
## Current Behavior

I don know if it was an update from windows or something else, but i usually only use my laptop for programing and nothing else.
## Steps to Reproduce

1.
2.
3.

## Your Environment

 

* Processing version:
* Operating System and OS version:
* Other information:

**Comment by Nematosis** [comment]
Could somebody help me or explain why this behaviour occur?

**Comment by Nematosis** [comment]
any ideas?

**Comment by Ikbenjijbent** [comment]
Uninstall then reinstall P3?  Update gpu drivers?

**Comment by Nematosis** [comment]
I solved the problem, and I think is a bug. Get this, while I was tripping on how to make my program I created a non-sense function in order to draw lines.

void Axis(){
  push();
  
  fill(200,15,15);
  
  line(-width/2,0,width/2,0);
  line(0,height/2,0,-height/2);
  
  for(int i = 0;i < (cenX*zoom);i ++){
    line(i*zoom,-10,i*zoom,10);
    line(-i*zoom,-10,-i*zoom,10);
  }
  for(int i = 0;i < (cenY*zoom);i ++){
    line(-10,-i*zoom,10,-i*zoom);
    line(-10,i*zoom,10,i*zoom);
  }
  pop();
}

So I noticed after a a while that this function even though it has no great  deal of operations (I know i could make it a lot more efficient... I was tripping lol) it is though enough to pull down the frame rate in processing from 60 to 2. 

So remmember, DON'T put any kind of operation within the logical test in a for cicle, processing can't handle it.
